infer-plugin/src/test/groovy/com/uber/infer/util/InferAndroidPluginAppIntegrationTest.groovy

```diff
@@ -46,7 +46,7 @@ class InferAndroidPluginAppIntegrationTest extends IntegrationTest {
                     }
 
                     dependencies {
-                        classpath 'com.android.tools.build:gradle:1.5.0'
+                        classpath 'com.android.tools.build:gradle:2.3.0-beta1'
```

Comment:
2.3.0 final has been released.

On rebase should update to `classpath 'com.android.tools.build:gradle:2.3.0'`